modyn/supervisor/internal/triggers/datadrifttrigger.py

```diff
@@ -232,7 +214,7 @@ def _evaluate_batch(
         if log:
             log.evaluations.append(drift_eval_log)
 
-        if not self.warmup_completed:
+        if not self.warmup_trigger.completed:
             # during the warmup phase we always want to reset the windows as if we detected drift;
```

Comment:
i just realized it (and know it wasnt touched by this pr) but maybe we should say why: otherwise the current and reference model would be very similar, right?

actually, this makes me wonder: since I use allow_overlap = True, I don't reset the model. do we expect the policies to work then at all?